### PR TITLE
noise profiles

### PR DESCRIPTION
**Pull request by rawfiner**
Fixes https://github.com/darktable-org/darktable/issues/12247
Fixes https://github.com/darktable-org/darktable/issues/12374

### CODE REVIEW

File: data/noiseprofiles.json [hunk 1]
```diff
@@ -913,6 +913,37 @@
             {"name": "Canon EOS 50D iso 12800", "iso": 12800, "a": [0.000591454854520338, 0.000275195972130737, 0.000445695275935004], "b": [1.51379068534984e-06, 1.61912116109517e-06, 7.02429683081879e-06]}
           ]
         },
+        {
+          "comment": "eos m50m2 contributed by user",
+          "model": "EOS M50 Mark II",
+          "profiles": [
+            {"name": "EOS M50m2 iso 100", "iso": 100, "a": [8.67963696322619e-06, 4.57517159107858e-06, 7.81889753444101e-06], "b": [2.40010265548342e-08, 1.37009371752072e-08, 2.2216741588009e-08]},
```
Comment (kmilos) [inline data/noiseprofiles.json:920]: Should we change the m2 to " Mark II" here to match displayed camera information??

Comment (TurboGit) [reply data/noiseprofiles.json:920]: Yes, probably better indeed even if name string is only for information.

Comment (rawfiner) [reply data/noiseprofiles.json:920]: Ok I'll do that

Comment (rawfiner) [reply data/noiseprofiles.json:920]: fixed in last push

**Review by TurboGit — APPROVED** [review]
Thanks!

**Pull request merged**